raiden/message_handler.py

```diff
@@ -70,6 +87,38 @@ def on_message(self, raiden: RaidenService, message: Message) -> None:
         else:
             log.error("Unknown message cmdid {}".format(message.cmdid))
 
+    @staticmethod
+    def handle_message_withdrawrequest(raiden: RaidenService, message: WithdrawRequest):
+        withdraw_request = ReceiveWithdrawRequest(
+            canonical_identifier=CanonicalIdentifier(
```

Comment:
is there a reason, why `message` can't provide the property `.canonical_identifier` ?

Reply:
message definitions are broken down into the separate attributes: `channel_id`, `token_network_address` ... etc and i haven't seen any of them providing this. So i followed what was done there.
Should i change this to have the `WithdrawRequest` message provide the `canonical_identifier` as a propery?

Reply:
your call :)